src/Offer/OfferRepository.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Offer;
+
+use Assert\AssertionFailedException;
+use Broadway\Domain\AggregateRoot;
+use Broadway\Repository\AggregateNotFoundException;
+use Broadway\Repository\RepositoryInterface;
+use CultuurNet\UDB3\Event\Event;
+use CultuurNet\UDB3\Event\EventRepository;
+use CultuurNet\UDB3\Place\Place;
+use CultuurNet\UDB3\Place\PlaceRepository;
+
+class OfferRepository implements RepositoryInterface
+{
+    /**
+     * @var EventRepository
+     */
+    private $eventRepository;
+
+    /**
+     * @var PlaceRepository
+     */
+    private $placeRepository;
+
+    public function __construct(
+        EventRepository $eventRepository,
+        PlaceRepository $placeRepository
+    ) {
+        $this->eventRepository = $eventRepository;
+        $this->placeRepository = $placeRepository;
+    }
+
+    public function save(AggregateRoot $aggregate): void
+    {
+        try {
+            $this->eventRepository->save($aggregate);
+        } catch (AssertionFailedException $e) {
+            $this->placeRepository->save($aggregate);
+        }
```

Comment:
Was thinking we would check the Aggregate class name ourselves, but nice that we can simply do this since the underlying repositories already do a check on the class name themselves! 👍